packages/terra-tag/src/Tag.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Tag.scss';
+
+const cx = classNames.bind(styles);
+
+const KEYCODES = {
+  TAB: 9,
+};
+
+const TagTheme = {
+  NORMAL: 'normal',
```

Comment:
keep as normal or light?

Reply:
On my tech design there was a discussion and we decided to go with Normal.